app/submitter/converter.py

```diff
@@ -88,7 +88,13 @@ def prepare_answers(metadata_store, questionnaire, answers):
             if item is not None:
                 value = answers[key]
                 if value is not None:
-                    data[item.code] = Converter._encode_value(value)
+                    if item.code not in data:
```

Comment:
After a quick discussion with @ajmaddaford it might makes sense to not change anything here until we know the format we send this to the downstream system in